projects/igniteui-angular/src/lib/combo/combo.component.spec.ts

```diff
@@ -2606,6 +2610,32 @@ describe('igxCombo', () => {
                 expect(selectionSpy).toHaveBeenCalledWith(expectedResults);
                 expect(input.nativeElement.value).toEqual(expectedDisplayText);
             });
+            it('should only select and display existing values in the data when programmatically selecting non existing items with ngModel', fakeAsync(() => {
```

Comment:
This one directly modifies the value of the field bound to the model and the one below changes the value through an API call. So, we should change the tests' descriptions accordingly.